programs/scholarship.md

```diff
@@ -5,28 +5,32 @@ hero-message: Pioneers in Engineering Alumni Scholarship
 ---
 
 # Pioneers in Engineering Alumni Scholarship
-Pioneers in Engineering strives to provide a quality STEM educational experience for Bay Area high school students. In keeping with our mission to promote science and engineering, we are happy to announce our seventh annual PiE alumni-funded scholarship for PiE students. The scholarship will be for a one-time amount of $2000.
+Pioneers in Engineering strives to provide a quality STEM educational experience for Bay Area high school students. In keeping with our mission to promote science and engineering, we are happy to announce our eighth annual PiE alumni-funded scholarship for PiE students. The scholarship will be for a one-time amount of $2000.
```

Comment:
Based on this, https://pierobotics.slack.com/archives/CJ82STX89/p1614875922055300

Let's go mention that finalists will get $500 this year

Reply:
Has the other docs been updated?

Reply:
I would like to hold off on making this change. We need to update a few other places.